### PR TITLE
Add list runtimes endpoint

### PR DESCRIPTION
**Pull request by kjaksik**
**Description**

Changes proposed in this pull request:

- Add list runtimes endpoint

**Related issue(s)**
#180

### CODE REVIEW

**Comment by kjaksik** [comment]
Please add subaccountRegion field to response

**Comment by kjaksik** [comment]
Also remove the deletedAt field and change the updatedAt during provisioning and upgrade

File: components/kyma-environment-broker/cmd/broker/main.go [hunk 1]
```diff
@@ -110,6 +111,7 @@ type Config struct {
 	}
 
 	TrialRegionMappingFilePath string
+	MaxPaginationPage          string `envconfig:"default=100"`
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/cmd/broker/main.go:114]: can we make it a string?

File: components/kyma-environment-broker/internal/runtime/handler.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
+package runtime
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/pkg/httphelpers"
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/runtime/handler.go:8]: minor; remove lines between those imports

File: components/kyma-environment-broker/internal/runtime/handler.go [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+func (h *Handler) getOperationsForInstance(instance internal.Instance) (*internal.ProvisioningOperation, *internal.DeprovisioningOperation, *internal.UpgradeKymaOperation, error) {
+	pOpr, err := h.operationsDb.GetProvisioningOperationByInstanceID(instance.InstanceID)
+	if err != nil && !dberr.IsNotFound(err) {
+		return nil, nil, nil, err
+	}
+	dOpr, err := h.operationsDb.GetDeprovisioningOperationByInstanceID(instance.InstanceID)
+	if err != nil && !dberr.IsNotFound(err) {
+		return nil, nil, nil, err
+	}
+	ukOpr, err := h.operationsDb.GetUpgradeKymaOperationByInstanceID(instance.InstanceID)
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/runtime/handler.go:95]: We can assume that there is at most one provisioning and one deprovisioning operation per instance. But it is nothing strange to see more than one upgrade operation per instance. Please review the storage operation

**Review by piotrmiskiewicz — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

File: components/kyma-environment-broker/README.md [hunk 4]
```diff
@@ -51,3 +51,4 @@ KEB binary allows you to override some configuration parameters. You can specify
 | **APP_PROVISIONING_MACHINE_IMAGE** | Defines the Gardener machine image used in a provisioned node. | None |
 | **APP_PROVISIONING_MACHINE_IMAGE_VERSION** | Defines the Gardener image version used in a provisioned cluster. | None |
 | **APP_TRIAL_REGION_MAPPING_FILE_PATH** | Defines a path to the file which contains a mapping between the platform region and the Trial plan region. | None |
+| **APP_MAX_PAGINATION_PAGE** | Defines the maximal number of objects that can be queried in one page using the endpoints that use pagination. | `100` |
```
Comment (klaudiagrz) [inline components/kyma-environment-broker/README.md:54]:
```suggestion
| **APP_MAX_PAGINATION_PAGE** | Defines the maximum number of objects that can be queried in one page using the endpoints that use pagination. | `100` |
```

**Review by klaudiagrz — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**